lib/langchain/vectorsearch/pinecone.rb

```diff
@@ -141,7 +141,7 @@ def similarity_search_by_vector(embedding:, k: 4, namespace: "", filter: nil)
     # @param filter [String] The filter to use
     # @yield [String] Stream responses back one String at a time
     # @return [String] The answer to the question
-    def ask(question:, namespace: "", filter: nil, &block)
+    def ask(question:, namespace: "", filter: nil, k: 4, &block)
       search_results = similarity_search(query: question, namespace: namespace, filter: filter)
```

Comment:
I think you meant to also pass the `k:` param to the `similarity_search(..., k: k)`

Reply:
🤦  yes - absolutely did mean to do that. long day sorry.